gtsam/navigation/CombinedImuFactor.h

```diff
@@ -123,7 +123,7 @@ struct GTSAM_EXPORT PreintegrationCombinedParams : PreintegrationParams {
  * it is received from the IMU) so as to avoid costly integration at time of
  * factor construction.
  *
- * @addtogroup SLAM
+ * @ingroup SLAM
```

Comment:
Can you please update the group to match the directory name?
E.g. for this file, it should be `@ingroup Navigation`.